src/Files.App/Helpers/DynamicDialogFactory.cs

```diff
@@ -224,5 +216,58 @@ public static DynamicDialog GetFor_CredentialEntryDialog(string path)
 
 			return dialog;
 		}
+
+		public static DynamicDialog GetFor_GitCheckoutConflicts(string checkoutBranchName, string headBranchName)
+		{
+			DynamicDialog dialog = null!;
+
+			var optionsListView = new ListView()
+			{
+				ItemsSource = new string[]
+				{
+					string.Format("BringChanges".GetLocalizedResource(), checkoutBranchName),
+					string.Format("StashChanges".GetLocalizedResource(), headBranchName),
+					"DiscardChanges".GetLocalizedResource()
```

Comment:
@ferrariofilippo what are your thoughts on adding a description for each item? It's pretty self-explanatory but it might be good to add them just in case.